lab_1/main.py

```diff
@@ -25,9 +51,18 @@ def remove_stop_words(tokens: list, stop_words: list) -> list:
     stop_words = ['the', 'is']
     --> ['weather', 'sunny', 'man', 'happy']
     """
-    pass
+    tokens_clean = []
+    if type(tokens) == list:
```

Comment:
Usually, there are check for input arguments and body function - on the same level.
No need for nesting.

Reply:
I don't understand...